flask/static/styles/website.css

```diff
@@ -189,7 +189,7 @@ footer {
   padding-left: 2vw; /* Space from the left edge of the screen */
   font-family: 'Work Sans', sans-serif;
   gap: 2vw;
-  font-size: 1.8vw;
+  font-size: 3vw;
```

Comment:
why are we using `vw` as a font size unit? I believe `rem` is more standard and more accessible (since rem will change w/ the browser font size, but vw wont)